dist/leaflet.css

```diff
@@ -52,6 +52,7 @@
 	max-height: none !important;
 	width: auto;
 	padding: 0;
+	mix-blend-mode: plus-lighter;

```

Comment:
Perhaps we can add a comment for future reference?
```suggestion
	/* See: https://bugs.chromium.org/p/chromium/issues/detail?id=600120 */
	mix-blend-mode: plus-lighter;

```

Reply:
My location doesn't show actually. How can I fix and update